Riot/Modules/Room/RoomViewController.m

```diff
@@ -1254,24 +1254,31 @@ - (void)setBubbleTableViewDisplayInTransition:(BOOL)bubbleTableViewDisplayInTran
 
 - (void)sendTextMessage:(NSString*)msgTxt
 {
+    // If the event occur on timeline not live, use the live data source to resolve event
+    BOOL isLive = self.roomDataSource.isLive;
+    if (isLive == NO) {
+        [self setupRoomDataSourceLive];
```

Comment:
We should probably add a check for whether self.roomDataSourceLive has already been set up here too, to prevent calling `setupRoomDataSourceLive` over and over if the user sends a few replies from the paginated timeline.